compiler/rustc_codegen_ssa/Cargo.toml

```diff
@@ -10,7 +10,7 @@ bitflags = "2.4.1"
 bstr = "1.11.3"
 # `cc` updates often break things, so we pin it here. Cargo enforces "max 1 semver-compat version
 # per crate", so if you change this, you need to also change it in `rustc_llvm`.
-cc = "=1.2.16"
+cc = "=1.2.33"
```

Comment:
We recently released find-msvc-tools for ssa, since we are informed that it only uses that part of the crate, could consider replacing cc with it

Reply:
Does that support Arm64EC well? 🤔

Reply:
The code is taken straight out of cc, it is a refactor I did that put the cc::windows_registry into a separate crate find-msvc-tools

So it would support anything cc already supports, so yes it would be

Reply:
Switched over.

`find-msvc-tools` already works correctly for Arm64EC, the change I needed in `cc` was for `rustc_llvm` to correctly build an Arm64EC copy of LLVM.